### PR TITLE
refactor(quarks): FRMW-405 Page and FRMW-406 Paginate filters

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-405 (`rxPaginate.Page`)
JIRA: https://jira.rax.io/browse/FRMW-406 (`rxPaginate.Paginate`)

- [x] Dev LGTM
- [x] Dev LGTM

---
Refactor `rxPaginate.Page` and `rxPaginate.Paginate` to Quark Filters.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `Page` Quark Filter

### Demo Page
screen shot 2016-02-09 at 10 26 18 am

### API Docs
screen shot 2016-02-09 at 10 27 30 am

**Comment by 100stacks** [comment]
## `Paginate` Quark Filter

### Demo Page
screen shot 2016-02-09 at 10 26 38 am


### API Docs
screen shot 2016-02-09 at 10 26 57 am

File: src/quarks/Page/scripts/Page.spec.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ describe('Pagination', function () {
         var page;
 
         beforeEach(function () {
+            module('encore.ui.quarks');
             module('encore.ui.rxPaginate');
```
Comment (100stacks) [inline src/quarks/Page/scripts/Page.spec.js]: We should be able to remove this after PR https://github.com/rackerlabs/encore-ui/pull/1534 merge.

Comment (100stacks) [inline src/quarks/Page/scripts/Page.spec.js]: We might be able to remove this after PR https://github.com/rackerlabs/encore-ui/pull/1534 merge.

File: src/quarks/Paginate/scripts/Paginate.spec.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ describe('Pagination', function () {
         var paginate, pager;
 
         beforeEach(function () {
+            module('encore.ui.quarks');
             module('encore.ui.rxPaginate');
```
Comment (100stacks) [inline src/quarks/Paginate/scripts/Paginate.spec.js]: We might be able to remove this after PR https://github.com/rackerlabs/encore-ui/pull/1534 merge.

**Comment by CITguy** [comment]
LGTM

**Comment by parlarjb** [comment]
LGTM

**Comment by 100stacks** [comment]
@parlarjb, rebased...much appreciated. :+1:

**Pull request merged**